addons/website/controllers/main.py

```diff
@@ -225,6 +225,27 @@ def social(self, social, **kwargs):
             raise werkzeug.exceptions.NotFound()
         return request.redirect(url)
 
+    @http.route('/website/get_suggested_link', type='json', auth="user", website=True)
+    def get_suggested_link(self, needle):
+        current_website = request.website
+        suggested = {}
+
+        if len(needle) < 3:
+            installed_controllers = current_website.get_suggested_controllers()
+            last_created_pages = current_website.get_website_pages(order='write_date desc', limit=5).mapped(lambda p: p.url)
```

Comment:
mapped('url')

quid finally about name/url ? instead of just url? You don't find it better?